src/plugins_types_bits.c

```diff
@@ -62,113 +68,123 @@ ly_type_store_bits(const struct ly_ctx *ctx, const struct lysc_type *type, const
     ret = ly_set_new(&items);
     LY_CHECK_GOTO(ret != LY_SUCCESS, cleanup_value);
 
-    ws_count = lws_count = 0;
-    for (index = 0; index < value_len; ++index) {
-        if (isspace(value[index])) {
-            ++ws_count;
-            continue;
+    /* get all values */
+    while (1) {
```

Comment:
the condition can be probably `index_end < value_len` to cover the most common case when the string ends right after a bit name. It would also avoid illusory infinite loops.